### PR TITLE
Updates README.md

### LINKED ISSUE #1: Update Readme.md
pls update asap!

### PR DESCRIPTION
**Pull request by a-jwc**
Resolves #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # hello courtney
-# helllooooo
+# helloooooooo
+
```
Comment (csr5321) [inline README.md:3]: why so many o's?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,9 @@
...
+
+# will be change 4
```
Comment (a-jwc) [inline README.md:5]: can this be change 5?

File: README.md (hunk 1 above)
Comment (a-jwc) [reply README.md:3]: more o's = more better

File: README.md (hunk 2 above)
Comment (a-jwc) [reply README.md]: ok

**Pull request merged**